### DISCUSSION TITLE
Unique Field

### DISCUSSION THREAD

**Discussion by ElliotHYLee** [pocketbase/pocketbase#2339] [Q&A] [answered]
Unless I'm missing something.. I can't find the unique field for v0.15

My usecase is to make a blog's slug as a unique url at the end. Is it moved to somewhere?

![image](https://user-images.githubusercontent.com/7537835/233751659-90686e2f-ff41-4f25-bbc7-8cf2a7fecd0a.png)

**Comment by juangiordana** [comment, marked as answer]
Check above the Cancel/Save changes buttons at the end of the form.
